PhysicsTools/PatAlgos/plugins/PATTriggerProducer.cc

```diff
@@ -558,7 +558,7 @@ void PATTriggerProducer::produce(Event& iEvent, const EventSetup& iSetup) {
       if (hltConfig.prescaleSize() > 0) {
         if (hltPrescaleProvider_.prescaleSet(iEvent, iSetup) != -1) {
           hltPrescaleTable = trigger::HLTPrescaleTable(
-              hltPrescaleProvider_.prescaleSet(iEvent, iSetup), hltConfig.prescaleLabels(), hltConfig.prescaleTable());
+              hltPrescaleProvider_.prescaleSet(iEvent, iSetup), hltConfig.prescaleLabels(), {});
```

Comment:
Here, I think the initialisation can be done with a dummy value, because information from the table is never actually used in this client. Not the greatest solution, but it works.

Using `prescaleTable<double/FractionalPrescale>` would not be possible here, because a map with `unsigned int`s is needed to initialise the `HLTPrescaleTable` instance. One could convert `prescaleTable<double/FractionalPrescale>` to a map with `unsigned int`s and feed the latter here to `HLTPrescaleTable`, but that's also clumsy. Between two sub-optimal options, I chose the one with less code.